File: PODSTAWY/hello.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-

ROK_TERAZ = 2017
ROK_PYTHON = 1991

def parzyste (n):
    ile = list(range(0, n+1,2))
    print(ile)
    return len(ile)

def main(args):
    imie = input ("Imię? ")
    wiek = int(input("Wiek?" ))
    rok_urodzenia = ROK_TERAZ - wiek
    print ("Witaj", imie)
    print ("Rok Urodzenia: ", rok_urodzenia)
    if ROK_PYTHON < rok_urodzenia:
        
        print ("Jestem starszy")
    elif ROK_PYTHON > rok_urodzenia:
        print ("Jestem młodszy")
    elif ROK_PYTHON == rok_urodzenia:
        print ("Tyle samo lat")
    
    n = int(input("Podaj liczbe naturalna: "))
    print('Ilość parzystych: ', parzyste(n))
    
  
  
  
  
    
int liczba, i, j, pierwiastek, *tablica;
cout << "Podaj maksimum: ";
cin >> liczba;
 
pierwiastek = (int)sqrt((double)liczba);
tablica = new int[liczba+1];
for (i = 0; i <= liczba; i++)
tablica[i] = i;
 
for (i = 2; i <= pierwiastek; i++)
{
if (tablica[i] != 0)
for (j = 2*i; j <= liczba; j+= i)
tablica[j] = 0;
}
 
cout << "Liczby pierwsze z przedzialu <2," << liczba << "> to:\n";
for (i = 2; i <= liczba; i++)
if (tablica[i] != 0)
cout << tablica[i] << " ";
cout << endl;
  
  
  
    
    
    return 0

if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))

File: PODSTAWY/hello.cpp
/*
 * hello.py
 */


#include <iostream>

using namespace std;

#define ROK_TERAZ 2017
#define ROK_CPP 1991

int parzyste (int n)
{
    int i;
    for (i=0; i<=n; i+=2)
        cout << i << " ";
    return i / 2;
}

int main(int argc, char **argv)

{   string tekst;
    int rok_urodzenia;
    int wiek;
    cout << "Jak masz na imie?" << endl;
    cin >> tekst;
    cout << "Witaj " << tekst << "!"<< endl;
    cout << "Ile masz lat?" << endl;
    cin >> wiek;
    cout << "Urodziłes sie w " << ROK_TERAZ-wiek << endl;
    
    rok_urodzenia = ROK_TERAZ-wiek;
    
    if (rok_urodzenia > ROK_CPP) {
        cout << "Jestem starszy!" << endl;
    }
    else if (rok_urodzenia < ROK_CPP) {
        cout << "Jestem młodszy" << endl;
    }
    else if (rok_urodzenia == ROK_CPP) {
        cout << "Mamy tyle samo lat" << endl;
    }
    
    cout << endl;
    
    int n;
    cout << "Podaj liczne naturalna: ";
    cin >> n;
    cout << endl;
    cout << "Ilosc parzystych: " << parzyste(n);
    
	
	return 0;
}

